app/models/chargeback_rate_detail.rb

```diff
@@ -195,6 +198,10 @@ def ambiguous_tier?(tier)
     tier.start == tier.finish
   end
 
+  def start_great_than_finish?(tier)
```

Comment:
WDYT about `start_greater_than_finish?`, or even `tier_continuity`? Further, did you consider making this a validation/method on `tier` itself?